dask/dataframe/methods.py

```diff
@@ -152,8 +152,11 @@ def describe_aggregate(values):
 def describe_numeric_aggregate(stats, name=None, is_timedelta_col=False):
     assert len(stats) == 6
     count, mean, std, min, q, max = stats
+    import sys
 
-    typ = pd.DataFrame if isinstance(count, pd.Series) else pd.Series
+    package_name = count.__class__.__module__.split(".")[0]
+    DD = sys.modules[package_name]
+    typ = DD.DataFrame if is_series_like(count) else DD.Series
```

Comment:
Perhaps the following instead? 

```python
typ = type(count.to_frame()) if is_series_like(count) else type(count)
```

This doesn't dive as deeply into internals

Reply:
Changed this part